File: Rust/FizzBuzz/fizzbuzz.rs
fn fizzbuzz(n: i32) {
    let mut i = 1;
    while i <= n {
        if i % 15 == 0 {
            println!("FizzBuzz");
        } else if i % 3 == 0 {
            println!("Fizz");
        } else if i % 5 == 0 {
            println!("Buzz");
        } else {
            println!("{}", i)
        }
        i += 1;
    }
}

fn main() {
    fizzbuzz(15);
}

File: Fortran/FizzBuzz/fizzbuzz.f
      subroutine fizzbuzz(n)
         do i = 1, n
            if(mod(i, 15) == 0) then
               write(*, fmt='(a)') 'FizzBuzz'
            else if(mod(i, 3) == 0) then
               write(*, fmt='(a)') 'Fizz'
            else if(mod(i, 5) == 0) then
               write(*, fmt='(a)') 'Buzz'
            else
               write(*, '(I0)') i
            end if
         end do
      end subroutine

      program main
         call fizzbuzz(15)
      end program main
